dbt_sugar/core/task/audit.py

```diff
@@ -0,0 +1,243 @@
+        return 0
+
+    def get_columns_tests_statistics_for_model(self) -> None:
+        """
+        Method to get the tests statistics from a model.
+
+        For the process a column is not tested if it does not have any tests.
+        """
+        columns = self.dbt_tests.get(self.model_name)
+
+        if not columns:
+            logger.info(f"Not able to get the test statistics for the model {self.model_name}")
+            return
+
+        # Init variables
+        model_number_columns = 0
+        model_columns_without_tests = 0
+        untested_columns = []
+
+        for column in columns:
+            tests = column["tests"]
+            model_number_columns += 1
+            if len(tests) == 0:
+                model_columns_without_tests += 1
+                untested_columns.append(column["name"])
+
+        percentage_not_tested_columns = round(
+            100 - ((model_columns_without_tests / model_number_columns) * 100), 2
+        )
+
+        # To print nice the data with the total coverage at the last column.
+        data = {
+            (column): (
+                str(percentage_not_tested_columns) if i == (len(untested_columns) - 1) else ""
+            )
+            for i, column in enumerate(untested_columns)
+        }
+
+        self.create_table(
+            title="Test Coverage", columns=["undocument columns", "coverage"], data=data
+        )
+
+    def get_columns_descriptions_statistics_for_model(self) -> None:
+        """
+        Method to get the descriptions statistics from a model.
+
+        For the process a column is not documented if the description is:
+        Not No description for this column.
+        """
+        documented_columns = self.get_documented_columns(
+            content=self.model_content,
+            model_name=self.model_name,
+        )
+        not_documented_columns = self.get_not_documented_columns(
+            content=self.model_content,
+            model_name=self.model_name,
+        )
+
+        number_documented_columns = len(documented_columns.keys())
+        number_not_documented_columns = len(not_documented_columns.keys())
+
+        total_number_columns = number_documented_columns + number_not_documented_columns
+        percentage_not_documented_columns = round(
+            100 - ((number_not_documented_columns / total_number_columns) * 100), 2
+        )
+
+        # To print nice the data with the total coverage at the last column.
+        data = {
+            (column): (
+                str(percentage_not_documented_columns)
+                if i == (len(not_documented_columns.keys()) - 1)
+                else ""
+            )
+            for i, column in enumerate(not_documented_columns.keys())
+        }
+
+        self.create_table(
+            title="Documentation Coverage", columns=["undocument columns", "coverage"], data=data
+        )
+
+    def create_table(self, title: str, columns: List[str], data: Dict[str, Any]) -> None:
```

Comment:
I think we might be able to be a bit stricter on the type hint of the data, do you expect a lot of variety?